cmd/kube-apiserver/app/server.go

```diff
@@ -443,7 +444,8 @@ func BuildGenericConfig(s *options.ServerRunOptions, proxyTransport *http.Transp
 		return nil, nil, nil, nil, nil, fmt.Errorf("invalid authentication config: %v", err)
 	}
 
-	genericConfig.Authorizer, genericConfig.RuleResolver, err = BuildAuthorizer(s, sharedInformers)
+	genericConfig.Authorizer, genericConfig.RuleResolver, err =
+		BuildAuthorizer(s, sharedInformers, serviceResolver, proxyTransport)
```

Comment:
missing dialer param, and missing conditional building of dialer using buildAuthorizationWebhookDialer, right?

Reply:
I think you were looking at an intermediate phase of the code as I was reworking it. Please stand by, I'll let you know when the change is ready for review.  Otherwise, you'll time looking at defunct code.

Do you get a new update when I change my new feature branch?  If so, I'll try to minimize that to remove disruption to your workflow. Sorry.